plugins/find/dialogs/find.js

```diff
@@ -678,7 +737,17 @@
 						label: lang.replaceWith,
 						isChanged: false,
 						labelLayout: 'horizontal',
-						accessKey: 'R'
+						accessKey: 'R',
+						onKeyDown: function( evt ) {
+							var keystroke = evt.data.getKeystroke();
+
+							if ( keystroke !== 13 ) {
```

Comment:
It would be good to avoid magic numbers and extract `13` to constant (like you did in the unit test).